scripts/job.pbs

```diff
@@ -55,13 +54,10 @@ main()
 	module load gaussian
 	g09 < "$TEMP_DIR/$PBS_JOBNAME.inp" > "$TEMP_DIR/$PBS_JOBNAME.out"
 
-	# log_echo "Formatting checkpoint file:"
-	# log_echo "  $FORMATTED_FILE"
-	# formchk "$CHECKPOINT_FILE" "$FORMATTED_FILE"
+	log_echo "Formatting checkpoint file:"
+	log_echo "  $FORMATTED_FILE"
```

Comment:
One last nitpick: should probably change `$FORMATTED_FILE` to `$TEMP_DIR/checkpoint.fck".
Probably better to say "Generating formatted checkpoint file:" too.
The current log messages make it seem like its creating it in the user dir, which is not actually the case. Later on in the teardown it logs where the formatted file is being copied to.